packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -54,6 +70,18 @@ export const setQueryTaggingDebounce = wireCommit('setQueryTaggingDebounce');
  */
 export const setSessionDuration = wireCommit('setSessionDuration');
 
+/**
+ * Tracks the tagging of the query using a debounce which ends if the user
+ * accepts a query.
+ *
+ * @public
+ */
+export const trackTaggingAction = moduleDebounce(
```

Comment:
Why is it named `trackTaggingAction`? This isn't an action, it is a wire.

```suggestion
export const trackTaggingWire = moduleDebounce(
```